package.json

```diff
@@ -25,14 +25,16 @@
     "react-router-dom": "^5.2.0",
     "sass": "^1.32.12",
     "sass-loader": "^11.0.1",
-    "style-loader": "^2.0.0"
+    "style-loader": "^2.0.0",
+    "styled-components": "^5.3.0"
```

Comment:
?? styled components?

Reply:
Added styled components in an atom earlier .